### PR TITLE
Create page 'Buy certificate'

### PR DESCRIPTION
**Pull request by rostyslaw27**
## Description

!!! This PR is only for UI part so tests will be at next PR where will be a logic of page !!!
!!! I add empty arrow function for onSubmit, it will be removed at next PR where will be a logic of page !!!

#### Screenshots

image

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/pages/gift-certificate/certificate-checkbox/certificate-checkbox.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const styles = useStyles();
+  const { t } = useTranslation();
+  const theme = getFromLocalStorage('theme');
+
+  const checkTheme = () => {
+    if (theme === LIGHT_THEME) {
+      return MATERIAL_UI_COLOR.PRIMARY;
+    }
+    return MATERIAL_UI_COLOR.DEFAULT;
+  };
```
Comment (byavdoshniak) [inline src/pages/gift-certificate/certificate-checkbox/certificate-checkbox.js:25]: we check our theme in palette that you are passing to makeStyles hook. 
And the question is: do we need to use inline styles in Checkbox component on the line 42 or we can set out styles in certificate-checkbox.styles file and use palette there?

File: src/pages/gift-certificate/gift-certificate.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  const { t } = useTranslation();
+  const styles = useStyles();
+  const email = useSelector(({ User }) => User?.userData?.email);
+  const [initialValues, setInitialValues] = useState('');
+
+  const [checkboxesArr, setCheckboxesArr] = useState([
+    { value: 500, checked: false, count: INITIAL_CERTIFICATE_COUNT },
+    { value: 1000, checked: true, count: INITIAL_CERTIFICATE_COUNT },
+    { value: 1500, checked: false, count: INITIAL_CERTIFICATE_COUNT }
+  ]);
```
Comment (byavdoshniak) [inline src/pages/gift-certificate/gift-certificate.js:22]: maybe we should put this array into a separate constant and pass it here

File: src/pages/gift-certificate/gift-certificate.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      e.checked = false;
+    });
+    setCheckboxesArr(checkboxesArr.map((e, key) => (key === index ? { ...e, checked: value } : e)));
+  };
+
+  const handleCountChange = (count, index) => {
+    setCheckboxesArr(checkboxesArr.map((e, key) => (key === index ? { ...e, count } : e)));
+  };
+
+  const checkboxContent = checkboxesArr.map((e, key) => (
```
Comment (byavdoshniak) [inline src/pages/gift-certificate/gift-certificate.js:47]: Rename please your mapping items. It.'s not obvious what you mean inside map and in your  handleCheckboxChange function. 
In my opinion, 'e' and 'key' should be 'checkbox'/'checkboxItem' and 'index' respectively.

File: src/pages/gift-certificate/gift-certificate.js [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    const certificate = t(`certificates.certificateRules.${index}`);
+    return (
+      <>
+        {`${index + 1}. ${certificate}`}
+        <br />
+      </>
+    );
+  };
+
+  const certificateRulesContent = certificateRules.map((e, key) => (
```
Comment (byavdoshniak) [inline src/pages/gift-certificate/gift-certificate.js:68]: same as comment above

**Review by byavdoshniak — CHANGES_REQUESTED** [review]

File: src/pages/gift-certificate/gift-certificate.js (hunk 2 above)
Comment (rostyslaw27) [reply src/pages/gift-certificate/gift-certificate.js]: Resolved

File: src/pages/gift-certificate/gift-certificate.js (hunk 3 above)
Comment (rostyslaw27) [reply src/pages/gift-certificate/gift-certificate.js:53]: Resolved

File: src/pages/gift-certificate/gift-certificate.js (hunk 4 above)
Comment (rostyslaw27) [reply src/pages/gift-certificate/gift-certificate.js]: Resolved

**Review by byavdoshniak — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**